spyder/plugins/editor/widgets/tests/test_introspection.py

```diff
@@ -764,6 +767,55 @@ def test_fallback_completions(fallback_codeeditor, qtbot):
     code_editor.toggle_code_snippets(True)
 
 
+@pytest.mark.slow
+@pytest.mark.first
+@flaky(max_runs=5)
+def test_kite_textEdit_completions(mock_codeeditor, qtbot):
```

Comment:
You forgot to change this to `mock_completions_codeeditor`.

Reply:
🤦‍♂